charts/openshift-etcd-backup/values.yaml

```
@@ -0,0 +1,92 @@
+# Default values for openshift-etcd-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+backup:
+  # Sub directory path
+  subdir: "/"
+  # Directory name of single backup
+  dirname: "+etcd-backup-%F-%H-%M-%S"
+  # expiretype could be days (keep backups newer than backup.keepdays,
+  # count (keep a number of backups with backup.keepcount),
+  # never (do not expire backups, keep all of them)
+  expiretype: "days"
+  # retention period
+  keepdays: "30"
+  # count retetion if expiretype set to count
+  keepcount: "10"
+  # backup schedule
+  schedule: "0 0 * * *"
+
+persistence:
+  # Set reclaim policy (Retain or Delete)
+  reclaimPolicy: Retain
+  # Define the storage size
+  capacity: 10Gi
+  nfs:
+    # Enable nfs backend storage
+    enabled: false
+    # NFS server name or IP
+    server: example.com
+    # NFS server path
+    path: "/etcd-backups"
+  hostPath:
+    # Enable hostPath
+    enabled: true
+    # hostPath existing path on host
+    path: "/opt/etcd-backups"
+  provisionned:
```

I would recommend to rename the variable from `provisionned` to `provisioning`. This would then also need to be updated in the templates.

```suggestion
  provisioning:
```